src/coreclr/vm/ecalllist.h

```diff
@@ -61,12 +61,12 @@
 
 
 FCFuncStart(gDependentHandleFuncs)
-    FCFuncElement("nInitialize",             DependentHandle::nInitialize)
-    FCFuncElement("nGetPrimary",             DependentHandle::nGetPrimary)
-    FCFuncElement("nGetPrimaryAndSecondary", DependentHandle::nGetPrimaryAndSecondary)
-    FCFuncElement("nFree",                   DependentHandle::nFree)
-    FCFuncElement("nSetPrimary",             DependentHandle::nSetPrimary)
-    FCFuncElement("nSetSecondary",           DependentHandle::nSetSecondary)
+    FCFuncElement("nInitialize",   DependentHandle::nInitialize)
+    FCFuncElement("nGetPrimary",   DependentHandle::nGetPrimary)
```

Comment:
It would be nice to rename these to GetTarget / GetDependent to have consitent names. (You can also drop the `n` prefix while you are on it - it should not be necessary.)

Reply:
Renamed all FCalls in 7a97865c4f2ee9b05aea8605d2b157e951000ac3 to follow the `Internal___` naming scheme 🙂